qlib/kernel/SignalDef.rs

```diff
@@ -109,6 +109,18 @@ pub struct PtRegs {
     pub __pad: u64,
 }
 
+#[cfg(target_arch = "aarch64")]
```

Comment:
Please drop this from the PR, as is already added in #1027

Reply:
@shrik3 Done.

Reply:
~> @shrik3 Done.~
Readded commits, as it was not covert on #1027.